docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/Calibrators/FixedPlatt.cs

```diff
@@ -45,28 +44,47 @@ public static void Example()
             // Transform the scored data with a calibrator transfomer by adding a new column names "Probability". 
             // This column is a calibrated version of the "Score" column, meaning its values are a valid probability value in the [0, 1] interval
             // representing the chance that the respective sample bears positive sentiment. 
-            var finalData = calibratorTransformer.Transform(scoredData).Preview();
-            PrintRowViewValues(finalData);
+            var finalData = calibratorTransformer.Transform(scoredData);
+            var outScoresAndProbabilities = mlContext.Data.CreateEnumerable<ScoreAndProbabilityValue>(finalData, reuseRowObject: false);
+            PrintScoreAndProbability(outScoresAndProbabilities, 5);
             // Score   4.18144   Probability 0.9856767
             // Score  -14.10248  Probability 7.890148E-07
             // Score   2.731951  Probability 0.9416927
             // Score  -2.554229  Probability 0.07556222
             // Score   5.36571   Probability 0.9955735
         }
 
-        private static void PrintRowViewValues(Microsoft.ML.Data.DataDebuggerPreview data)
+        private static void PrintScore(IEnumerable<ScoreValue> values, int rows)
         {
-            var firstRows = data.RowView.Take(5);
+            foreach (var value in values)
+            {
+                if (rows-- <= 0)
+                    break;
 
-            foreach (Microsoft.ML.Data.DataDebuggerPreview.RowInfo row in firstRows)
+                Console.WriteLine("{0, -10} {1, -10}", "Score", value.Score);
+            }
+        }
+
+        private static void PrintScoreAndProbability(IEnumerable<ScoreAndProbabilityValue> values, int rows)
+        {
+            foreach (var value in values)
             {
-                foreach (var kvPair in row.Values)
-                {
-                    if (kvPair.Key.Equals("Score") || kvPair.Key.Equals("Probability"))
-                        Console.Write($" {kvPair.Key} {kvPair.Value} ");
-                }
-                Console.WriteLine();
+                if (rows-- <= 0)
+                    break;
+
+                Console.WriteLine("{0, -10} {1, -10} {2, -10} {3, -10}", "Score", value.Score, "Probability", value.Probability);
             }
```

Comment:
please replace with this simpler version:

foreach (var value in values.Take(rows))
    Console.WriteLine(...)